### ISSUE TITLE
Changing default branch back to master

### ISSUE THREAD

**Issue by Ad5001** [flathub/flathub#4849]
Hello,

I've recently noticed that by default branch of LogarithmPlotter (https://github.com/flathub/eu.ad5001.LogarithmPlotter) has been set to an old branch (branch/0.3.0) in Discover and GNOME Software by default. Is there a way to revert it back to master? And if not, could that old branch be deleted?

Thanks in advance,

**Comment by hfiguiere** [comment]
![image](https://github.com/flathub/flathub/assets/114441/7964a9e0-f4ec-43f8-9dce-7433a8e7e13e)

either somebody did change and not comment or it's already the case.

**Comment by hfiguiere** [comment]
btw I don't think it would have been changed by an admin because the use of branch isn't something expected.

**Comment by bbhtt** [comment]
I think they are saying that the branch is branch/0.3.0 in GNOME software and discover. Not exactly sure what that means. Can you post a picture?

**Comment by hfiguiere** [comment]
It's likely not a flathub problem.

Note that flatpak doesn't handle branch as something that might switch over, and if it was created on the package it's a mistake.

**Comment by Ad5001** [comment]
@bbhtt Exactly. Discover only provides branch/0.3.0 (old version) 
![Screenshot_20240111_191315](https://github.com/flathub/flathub/assets/16436883/b1c8c2c4-2769-4274-9a69-24151f167dff)

Meanwhile, GNOME Software uses branch/0.3.0 by default.
![Screenshot_20240111_191639](https://github.com/flathub/flathub/assets/16436883/057a5a3e-8e84-4919-9628-b6ea86240568)

@hfiguiere I do know that on my flathub repo the default branch is master, and I can see the latest version on flathub.org. Does that mean the issue lies with the flatpak client and neither flathub nor the Discover or GNOME Software?

(I sincerely apologize if my botched English is causing some confusion)

**Comment by hfiguiere** [comment]
![image](https://github.com/flathub/flathub/assets/114441/bbce7911-1bec-47d7-abb0-90313d009962)

**Comment by Ad5001** [comment]
Exactly, by default it uses branch/0.3.0 (see New in Version 0.3.0 and permissions Unsafe vs Safe on master). If flathub doesn't have any control on which branch is selected by default, do you know who I should take the issue to?

**Comment by hfiguiere** [comment]
I guess @barthalion will need to yank that branch.

**Comment by hfiguiere** [comment]
Filed https://gitlab.gnome.org/GNOME/gnome-software/-/issues/2420 for GNOME Software

**Comment by bbhtt** [comment]
branch/3.0 should be gone https://github.com/flathub/flathub/issues/5757, you might need to restart gnome software / clear ~/.cache/gnome-software and run flatpak update to see the effect.
